readthedocs-theme/templates/article.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+{% block html_lang %}{{ article.lang }}{% endblock %}
+
+
+{% block title %}{{ article.title|striptags  }} - {{ SITENAME }}{% endblock %}
+
+
+{% block content %}
+
+
+  <!-- Article -->
+  <section class="ui vertical segment padded" id="article">
+    
+    <div class="ui container">
+      <div class="ui grid centered">
+        <div class="column twelve wide computer fourteen wide tablet sixteen wide mobile">
+
+
+          <h1 class="ui header">
+            <span class="ui header">{{ article.title }}</span>
+          </h1>
+
+          <div class="ui basic vertical segment clearing">
+            <div class="ui fitted divider"></div>
+            <div class="ui basic vertical segment left floated">{% for author in article.authors %}{{ author }}{% if not loop.last %}, {% endif %}{% endfor %}</div>
+            <div class="ui basic vertical segment right floated"><time datetime="{{ article.date.isoformat() }}">{{ article.locale_date }}</time></div>
+          </div>
+
+          <img class="ui fluid image" src="{{ article.image or '/theme/img/defaults/alesia-kazantceva-VWcPlbHglYc-unsplash.jpg' }}">
```

Comment:
Probably makes more sense renamed to a generic image name, so that we can easily replace it in the future.

Also, `readthedocsext/theme` is the output path of webpack, it might make sense to keep all of the blog post header images together in a single path

```suggestion
          <img class="ui fluid image" src="{{ article.image or '/images/posts/default.png' }}">
```

Reply:
I agree with the image name. I used the original so we could keep track of the source, but since we are using a full attribution that's not needed. I'll also move it to the blog `images/posts/` folder.

Reply:
Looking further at the [Pelican structure](https://docs.getpelican.com/en/latest/themes.html#structure) for themes I notice that images inside the folder `/content/images/posts/` are only built if they are referenced from a content file. Images that are used _(only)_ by the templates must be placed inside the template folder structure, which is handled by Webpack.

Reply:
I didn't have this issue. You might find the images are not copied immediately when using `npm run dev` though. Saving a template file does copy files from `/images/posts/default.png` to the output path. So, I think `/images/posts/` is still the correct path.

Reply:
https://github.com/readthedocs/site-community/pull/51/commits/f22cd45de8a94c54a127c141c2fdda4c8621637e moves the default image to the `/content/images/posts/` folder.

I'm still not getting the correct build thought.
Running `npm run build` then `npm run dev`: the `test.jpg` image, that is referenced from the `.md` and `.rst` files, is built into the `/output` folder but the `default.jpg` is not. Tried several other things with always the same result.

Reply:
Seems you didn't rebase on d7990011d065e321d9fffd3508a75bc865876d5a after merging, so none of the images in this path would get picked up at all.

Reply:
I pushed up eeea54d to resolve this.

Reply:
Loads correctly now: 
![image](https://user-images.githubusercontent.com/1140183/154564116-cc4c29a8-c1f5-4466-aeab-88e161c2130a.png)